scipy/optimize/_minimize.py

```diff
@@ -202,6 +204,17 @@ def minimize(fun, x0, args=(), method=None, jac=None, hess=None,
     and either the Hessian or a function that computes the product of
     the Hessian with a given vector.
 
+    Method :ref:`trust-iterative <optimize.minimize-trustiterative>`
+    uses a iterative trust-region algorithm described in [13]_ for
```

Comment:
As I mentioned before this line can be changed to "uses a (near) exact algorithm to solve the trust-region subproblem" (or something like that).